Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -138,8 +138,42 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
         /// <returns>The project relative path.</returns>
         /// <remarks>This doesn't produce paths that contain step out '..' relative paths.</remarks>
         public static string GetAssetDatabasePath(string absolutePath)
+        {
+            string assetDatabasePath = Path.GetFullPath(absolutePath).Replace("\\", "/");
+            string token = string.Empty;
+            string newRoot = string.Empty;
+            if (assetDatabasePath.Contains("Assets"))
```

Comment:
There's an edge case I found that this does not work for:

1. Texture you are looking for is under "StandardAssets" subfolder, under the "PackageCache" folder.

this set of if statements sets the token to be "assets" in taht case.

Reply:
Good catch! Thanks @julenka! I'll add a fix (and test case)

Reply:
fixed and new edge case tests added